assemblyline/odm/models/config.py

```diff
@@ -1533,6 +1533,8 @@ class ExternalLinks(odm.Model):
         description="If the classification of the item is higher than the max_classificaiton, can we let the user "
                     "bypass the check and still query the external link?")
     name: str = odm.Keyword(description="Name of the link")
+    encoding: str = odm.Enum(values=["url", "sha256"], default="url",
+                                description="How should the target value be encoded (used with double_encode configuration)")
     double_encode: bool = odm.boolean(default=False, description="Should the replaced value be double encoded?")
```

Comment:
I may be clearer to modify the description with something along the lines of
> Should the replaced value be encoded a second time using url encoding?
We wouldn't want people thinking we are going to sha256 twice if they use sha256 and double_encode.